android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ItemTouchHandler.java

```diff
@@ -39,6 +39,7 @@ public class ItemTouchHandler extends ItemTouchHelper.SimpleCallback
 	private int moveEndIndex = -1;
 	private Drawable icon;
 	private final ColorDrawable background;
+	private boolean hasTouchStarted = false;
```

Comment:
Should this be named `hasMoveStarted`? Since it tracks when `MotionEvent.ACTION_MOVE` occurs, not `MotionEvent.ACTION_DOWN`, which would be the initial touch.

Reply:
Makes sense, thx!